GameOfLife/Calculations/CellPopulator.cs

```diff
@@ -0,0 +1,51 @@
+﻿namespace GameOfLife
+{
+
+    public class CellPopulator
+    {
+        /// <summary>
+        /// Pupulates array with random numbers either 1 or 0. Live cell = 1. Dead
+        /// cell = 0.
+        /// </summary>
+        /// <param name="gameField"></param>
```

Comment:
Dont leave parameter summaries empty:
`<param name="gameField">Here needs some text explaining parameter, like, Game field containing the cells.</param>`

Reply:
Also, empty `<returns></returns>` tags can be removed, if you dont include anything in them

Reply:
Check in all files.